README.md

```diff
@@ -13,32 +13,32 @@ Add analytics to your Ruby and Rails apps and gems with the **[Snowplow][snowplo
 
 With this tracker you can collect event data from your **[Ruby][ruby]** applications, **[Ruby on Rails][rails]** web applications and **[Ruby gems][rubygems]**.
 
-## Quickstart
+## Maintainer Quickstart
 
-Assuming git, **[Vagrant][vagrant-install]** and **[VirtualBox][virtualbox-install]** installed:
+Clone this repo. To run the tests locally, you will need [Docker][docker] installed:
 
 ```bash
- host$ git clone https://github.com/snowplow/snowplow-ruby-tracker.git
- host$ cd snowplow-ruby-tracker
- host$ vagrant up && vagrant ssh
-guest$ cd /vagrant
-guest$ gem install bundler
-guest$ bundle install
-guest$ rspec
+docker build . -t ruby-tracker
+docker run -d ruby-tracker
```

Comment:
We don't need to run this in detached mode (`-d`) as once `rspec` returns, the container will close.

Reply:
Just had an extra thought, it might also be worth mentioning around the docker instructions that if updating the Gemfile, you'll need to rebuild the docker image.

Reply:
Just had an extra thought, it might also be worth mentioning around the docker instructions that if updating the Gemfile, you'll need to rebuild the docker image.